src/Stomp/Client.php

```diff
@@ -154,6 +160,20 @@ public function setVhostname($host = null)
         $this->host = $host;
     }
 
+  /**
+   * Set the desired heartbeat for the connection.
```

Comment:
we should spent a small sentence why one would want to use a heartbeat (which problem it solves conceptionally)